app/controllers/api/users_controller.rb

```diff
@@ -1,17 +1,31 @@
 class API::UsersController < API::RestfulController
```

Comment:
I reckon it makes more sense to change it now, before it's merged into master.. it's only gonna be harder later on isn't it?